pkg/common/cns-lib/volume/manager.go

```diff
@@ -59,6 +59,7 @@ const (
 	VolumeOperationTimeoutInSeconds = 300
 
 	listviewAdditionError = "failed to add task to list view"
+	timeoutError          = "time out for task"
```

Comment:
Is this an error message from the VC ?
This error message is only emitted when vCenter server language is set to en_US.
What if vCenter server language is different ?